src/vuex-getters.md

```diff
@@ -89,12 +89,12 @@ describe("poodlesByAge", () => {
 
 Instead of actually passing the real `poodles` getter, we pass in the result it would return. We already know it is working, since we wrote a test for it. This allows us to focus on testing the logic unique to `poodlesByAge`.
 
-It is possible to have `async` getters. They can be tested using the same technique are `async` actions, which you can read about [here](https://lmiller1990.github.io/vue-testing-handbook/vuex-actions.html).
+It is possible to have `async` getters. They can be tested using the same technique as `async` actions, which you can read about [here](https://lmiller1990.github.io/vue-testing-handbook/vuex-actions.html).
 
 ## Conclusion
 
-- `getters` are just plain JavaScript functions
-- when testing `getters` in isolation, you need to pass the state manually
-- if a getter uses another getter, you should stub the expected return result of the first getter. This will give you more fine grained control over the test, and let you focus on testing the getter in question
+- `getters` are just plain JavaScript functions.
+- When testing `getters` in isolation, you need to pass the state manually.
+- If a getter uses another getter, you should stub the expected return result of the first getter. This will give you more fine grained control over the test, and let you focus on testing the getter in question
```

Comment:
Minor changes here as well. You should be consistent with your list items. Whether you go punctuation/no punctuation, do it for every item in the list. Hope this is helpful.